docs/references/subcommands/analyze.md

```diff
@@ -71,20 +71,28 @@ We support the following archive formats:
   - `xz` compression
   - `zstd` compression
 
+### Enabling additional strategies
+
+In addition to the [standard flags](#specifying-fossa-project-details), the analyze command supports the following additional strategy flags:
+
+| Name                              | Description                                                                                                                                                              |
+|-----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--detect-vendored`               | Enable the vendored source indentification engine. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md).                           |
```

Comment:
Should the flags in the `Name` column link directly to `./detect-vendored.md` and `./detect-dynamic.md`?

Reply:
Ooh, good idea